.travis.yml

```diff
@@ -95,7 +95,7 @@ matrix:
       compiler: clang
       script:
         - set -e
-        - pip install --user pre-commit-hooks
+        - python2 -m pip install --user --cache-dir=$PWD/cache pre-commit-hooks
```

Comment:
`--with-python=2` is already hard-coded in travis.yml, so I think it was a good idea to use `python2` explicitly for all jobs. 

Why did you add cache-dir to the `autotools-clang` and `cmake-submodules-and-pytest` jobs?

Reply:
It seemed like a non-destructive move, although it is not necessarily needed. I can remove it, if you insist.

Reply:
Thanks. I don't, I was just curious.